Remotion/Web/Core/ExecutionEngine/UrlMapping/UrlMappingConfiguration.cs

```diff
@@ -43,7 +43,10 @@ public class UrlMappingConfiguration: ConfigurationBase
 
   public static UrlMappingConfiguration CreateUrlMappingConfiguration (string configurationFile)
   {
```

Comment:
since we're touching here anyway and removing extra config stuff: please check, I think we can inline the base class ConfigurationBase.